src/Categories/Category/Instance/SimplicialSet/Properties.agda

```diff
@@ -0,0 +1,212 @@
+    proj₁ (hom (lower b)) (proj₁ f x)  ≡⟨ lower (eq (proj₁ f x)) ⟩
+    proj₁ (hom (lower b′)) (proj₁ f x) ≡⟨ cong (proj₁ (hom (lower b′))) (f≈g x) ⟩
+    proj₁ (hom (lower b′)) (proj₁ g x) ∎
+  } 
+  where
+    open Boundary
+    open Eq
+    open ≡-Reasoning
+
+--------------------------------------------------------------------------------
+-- Horns
+-- 
+-- The idea here is essentially the same as the boundaries, but we exclude the kth
+-- face map as a possible factor.
+
+record Horn (m n-1 : ℕ) (k : Fin (ℕ.suc n-1)) : Set where
+  field
+    horn : Boundary m n-1
+
+  open Boundary horn public
+
+  field
+    is-horn : factor-dim Eq.≢ k
+
+Λ[_,_] : (n : ℕ) → Fin n → ΔSet
+Λ[ ℕ.suc n , k ] = record
+  { F₀ = λ m → record
+    { Carrier = Lift o (Horn m n k)
+    ; _≈_ = λ (lift b) (lift b′) → ∀ x → Lift ℓ (proj₁ (hom b) x ≡ proj₁ (hom b′) x)
+    ; isEquivalence = record
+      { refl = λ x → lift refl
+      ; sym = λ eq x → lift (sym (lower (eq x)))
+      ; trans = λ eq₁ eq₂ x → lift (trans (lower (eq₁ x)) (lower (eq₂ x)))
+      }
+    }
+  ; F₁ = λ f → record
+    { _⟨$⟩_ = λ (lift h) → lift $ record
+      { horn = boundary-map f (horn h)
+      ; is-horn = is-horn h
+      }
+    ; cong = λ eq x → eq (proj₁ f x)
+    }
+  ; identity = λ eq → eq
+  ; homomorphism = λ {_} {_} {_} {f} {g} eq x → eq (proj₁ f (proj₁ g x))
+  ; F-resp-≈ = λ {_} {_} {f} {g} f≈g {b} {b′} eq x → lift $ begin
+    proj₁ (hom (lower b)) (proj₁ f x)  ≡⟨ lower (eq (proj₁ f x)) ⟩
+    proj₁ (hom (lower b′)) (proj₁ f x) ≡⟨ cong (proj₁ (hom (lower b′))) (f≈g x) ⟩
+    proj₁ (hom (lower b′)) (proj₁ g x) ∎
+  }
+  where
+    open Horn
+    open Eq
+    open ≡-Reasoning
+
+
+--------------------------------------------------------------------------------
+-- Kan Complexes
+--
+-- These are technically "Algebraic" Kan Complexes, as they come with a choice of fillers
+-- However, this notion is far easier than the more geometric flavor,
+-- as we can sidestep questions about choice.
+
+module _ where
+  open Category (SimplicialSet o ℓ)
+
+  -- Inclusion of boundaries
+  ∂Δ-inj : ∀ {n} → ∂Δ[ n ] ⇒ Δ[ n ]
+  ∂Δ-inj {ℕ.zero} = ntHelper record
+    { η = λ X → record
+      { _⟨$⟩_ = ⊥-elim
+      ; cong = λ { {()} }
+      }
+    ; commute = λ { _ {()} _ }
+    }
+  ∂Δ-inj {ℕ.suc n} = ntHelper record
+    { η = λ X → record
+      { _⟨$⟩_ = λ (lift b) → lift (hom b)
+      ; cong = λ eq → lift (λ x → lower (eq x))
+      }
+    ; commute = λ f eq → lift (λ x → lower (eq (proj₁ f x)))
+    }
+    where
+      open Boundary
+
+  -- Inclusion of n-horns into n-simplicies
+  Λ-inj : ∀ {n} → (k : Fin n) → Λ[ n , k ] ⇒ Δ[ n ]
+  Λ-inj {n = ℕ.suc n} k = ntHelper record
+    { η = λ X → record
+      { _⟨$⟩_ = λ (lift h) → lift (hom h)
+      ; cong = λ eq → lift (λ x → lower (eq x))
+      }
+    ; commute = λ f eq → lift (λ x → lower (eq (proj₁ f x)))
+    }
+    where
+      open Horn
+  
+  record IsKanComplex (X : ΔSet) : Set (o ⊔ ℓ) where
```

Comment:
This is not the right file in which to define this. An actual `KanComplex` file would make sense. The top-level is getting a little crowded though. `Categories.Constructions.KanComplex` ?

Reply:
Good call! Moved